### PR TITLE
add some improvements

### PR DESCRIPTION
**Pull request by guilhermebodin**

### CODE REVIEW

File: src/StateSpaceModels.jl [hunk 1]
```diff
@@ -119,7 +119,7 @@ export kalman_smoother
 export loglike
 export num_states
 export number_hyperparameters
-export results
+export print_results
```
Comment (raphaelsaavedra) [inline src/StateSpaceModels.jl:122]: This is a breaking change, so the version should be bumped to `0.6.0` – alternatively we could simply not rename `results` to `get_results`, keep exporting it, and also export `print_results`

File: src/fit.jl [hunk 2]
```diff
@@ -29,6 +30,7 @@ function fit!(
     model::StateSpaceModel;
     filter::KalmanFilter=default_filter(model),
     optimizer::Optimizer=default_optimizer(model),
+    save_hyperparameter_distribution_results::Bool=true
```
Comment (raphaelsaavedra) [inline src/fit.jl:33]: Can we use a shorter kwarg for simplicity? Maybe just `save_hyperparameter_distribution`?

**Review by raphaelsaavedra — CHANGES_REQUESTED** [review]

**Comment by guilhermebodin** [comment]
I agreed with the comments. Right now I prefer to release a 0.6

**Comment by guilhermebodin** [comment]
This should close #193

**Pull request merged**